src/components/tag/Tag.root.tsx

```diff
@@ -0,0 +1,63 @@
+import styled, { css } from 'styled-components';
+import { FlattenSimpleInterpolation, ThemedStyledProps } from 'styled-components/ts3.6';
+
+import { Spacing } from '../../foundations/layout';
+import { applyPadding } from '../../foundations/layout/Grid.mixins';
+import { applyShape } from '../../foundations/shape/Shape.mixins';
+import { TTheme } from '../../foundations/theme-provider/themes/theme.types';
```

Comment:
nit and not related to this PR: we need to unify the im-/exports to always import from the index file. and I would rename the `foundations/theme-provider` to `foundation/theme`